metadata-ingestion/src/datahub/ingestion/source/delta_lake/source.py

```diff
@@ -161,10 +162,12 @@ def _create_operation_aspect_wu(
             )
 
             operation_custom_properties = dict()
-            for key, val in hist.items():
+            for key in sorted(hist.keys()):
```

Comment:
This will work just fine:
```suggestion
            for key, value in sorted(hist.items()):
```